charts/grafana-dashboards/trivy-teams/trivy-teams.json

```diff
@@ -24,7 +24,6 @@
   "description": "Dashboard to view trivy published metrics\r\nhttps://aquasecurity.github.io/trivy-operator/v0.1.5/integrations/metrics/",
   "editable": true,
   "fiscalYearStartMonth": 0,
-  "gnetId": 16652,
```

Comment:
I can't see this dashboard in a team
<img width="1889" alt="image" src="https://github.com/redkubes/otomi-core/assets/17126497/d778810c-02e7-4842-a7a6-fb215beab0ca">

P.S.
Tracy has been enabled